src/WebJobs.Extensions.DurableTask.Analyzers/DurableFunctionsCodeFixProvider.cs

```diff
@@ -12,17 +12,14 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Analyzers
 {
     public abstract class DurableFunctionsCodeFixProvider : CodeFixProvider
     {
-        protected async Task<Document> ReplaceWithIdentifierAsync(Document document, SyntaxNode identifierNode, CancellationToken cancellationToken, string expression)
+        protected async Task<Document> ReplaceWithIdentifierAsync(Document document, SyntaxNode identifierNode, CancellationToken cancellationToken, string identifierString)
```

Comment:
I'm not sure I understand what we gain from this being in an abstract class, as opposed to just being a static method. Do we use polymorphism to change behavior, or does it use any `DurableFunctionsCodeFixProvider`-specific state? From what I can tell it does not.

Reply:
It doesn't have state. Good point, I'll make it a static class.